docs/api/proposed_endpoints.md

```diff
@@ -0,0 +1,210 @@
+[TOC]
+
+# Posts related actions
+
+## Create a post
+
+Create a post of contents.
+
+```
+POST /api/posts (requires authentication)
+```
+
+**Parameters**
+
+Name          | Data Type   | Description
+--------------|-------------|------------------------------------------
+id            | UUID        | identity of the post
+posted_by     | UUID        | Foreign key to `Users` model
+posted_at     | text        | social media ploatform to post
```

Comment:
This should be `posted_on` everywhere. `posted_at` sounds more like it will be a `datetime`.